exercises/acronym/acronym_test.py

```diff
@@ -31,6 +31,9 @@ def test_consecutive_delimiters(self):
         self.assertEqual(
             abbreviate('Something - I made up from thin air'), 'SIMUFTA')
 
+    def test_apostrophes(self):
+        self.assertEqual(abbreviate('Halley\'s Comet'), 'HC')
```

Comment:
For readability, can you please use double quotes around the input string to negate the need for escaping the apostrophe?